src/guides/v2.3/frontend-dev-guide/validations/custom-form-validation.md

```diff
@@ -0,0 +1,119 @@
+---
+group: frontend-developer-guide
+title: Custom form validation
+contributor_name: Atwix
+contributor_link: https://www.atwix.com/
+---
+
+This tutorial shows you how to validate your own form, using `mage/validation` library, before submitting it to the server.
+This gives the possibility to customer to fix all the invalid data straight away, which improves the good user experience.
```

Comment:
As a rule, we don't need to improve a good experience. Unless we are going to make it perfect 🙂

```suggestion
This gives the possibility to customer to fix all the invalid data straight away, which improves the user experience and user interface accessibility.
```